lms/static/js/views/fields.js

```diff
@@ -288,7 +319,7 @@
             },
 
             updateValueInField: function () {
-                this.$('.u-field-value ').html(_.escape(this.modelValue()));
+                HtmlUtils.setHtml(this.$('.u-field-value '), HtmlUtils.ensureHtml(this.modelValue()));
```

Comment:
Since this was escaping earlier, you can assume this is always text, and you can go with the simpler:
```
this.$('.u-field-value ').text(this.modelValue());
```
Other tips:
1. You don't need `ensureHtml()` with `setHtml()`.  Ensure is built into all the other HtmlUtils functions.  
2. An example of where you might use ensure would be if it was the only HtmlUtils function you were using.  The following is basically equivalent to using setHtml():
```
this.$('.u-field-value ').html(HtmlUtils.ensureHtml(this.modelValue())));
```